browser/extensions/brave_component_loader.cc

```diff
@@ -18,7 +18,7 @@
 #include "brave/components/constants/brave_switches.h"
 #include "brave/components/constants/pref_names.h"
 #include "brave/components/web_discovery/buildflags/buildflags.h"
-#include "brave/components/web_discovery/common/util.h"
+#include "chrome/browser/browser_process.h"
```

Comment:
was this just an unrelated missing header?

Reply:
hmm. i don't think so. let me revert that change